astropy/io/ascii/core.py

```diff
@@ -36,6 +36,104 @@
 # Similar dictionary for fast readers
 FAST_CLASSES = {}
 
+
+class CsvWriter(object):
```

Comment:
I wonder if this couldn't be implemented through the normal machinery of the `csv` module by implementing a new Dialect.  

That said, this seems fine too and I don't have strong feelings about it either way.